### PR TITLE
Shadow Rift item drop penalty

### PR DESCRIPTION
**Pull request by gausie**
Shadow Rift divides item drop bonuses by 5 for all fights that take place there, which needs support by mafia. We previously only dealt in "doublers" which I have renamed to "multipliers" to reflect this.

Also added some spaded item drops to the shadow monsters, and some missing elemental attacks to the shadow prism

### CODE REVIEW

File: test/net/sourceforge/kolmafia/ModifiersTest.java [hunk 1]
```diff
@@ -377,13 +378,24 @@ public void bendinHellAndDirtyPearStack() {
         assertThat(mods.getDouble(DoubleModifier.SLEAZE_SPELL_DAMAGE), equalTo(200.0));
       }
     }
+
+    @Test
+    void shadowRiftQuartersItemDrop() {
```
Comment (midgleyc) [inline test/net/sourceforge/kolmafia/ModifiersTest.java:383]: `fifths`?

**Review by midgleyc — CHANGES_REQUESTED** [review]

File: test/net/sourceforge/kolmafia/ModifiersTest.java (hunk 1 above)
Comment (gausie) [reply test/net/sourceforge/kolmafia/ModifiersTest.java:383]: whoopsy

**Review by midgleyc — APPROVED** [review]
I assume you have confirmed that this appears in `modtrace` as expected?

**Comment by gausie** [comment]
> I assume you have confirmed that this appears in `modtrace` as expected?

I have! I even accidentally left some code to that effect in an earlier commit.

**Pull request merged**